tensorflow/tsl/mkl/build_defs.bzl

```
@@ -102,7 +102,8 @@ def mkl_deps():
     """
     return select({
         "@org_tensorflow//tensorflow/tsl/mkl:build_with_mkl_aarch64": ["@mkl_dnn_acl_compatible//:mkl_dnn_acl"],
-        "@org_tensorflow//tensorflow/tsl:linux_x86_64": ["@mkl_dnn_v1//:mkl_dnn"],
+        "@org_tensorflow//tensorflow/tsl:linux_x86_64_with_onednn_v2": ["@mkl_dnn_v1//:mkl_dnn"],
+        "@org_tensorflow//tensorflow/tsl:linux_x86_64_with_onednn_v3": ["@onednn_v3//:mkl_dnn"],
```

Can we just do this? This way, we won't need the `build_with_onednn_v2` config.
```suggestion
        "@org_tensorflow//tensorflow/tsl:linux_x86_64_with_onednn_v3": ["@onednn_v3//:mkl_dnn"],
        "@org_tensorflow//tensorflow/tsl:linux_x86_64": ["@mkl_dnn_v1//:mkl_dnn"],
```

---

@penpornk I tried removing `build_with_onednn_v2` config (per your suggestion), but oneDNN v3.x fails to build with the following error:
```
ERROR: tensorflow/core/kernels/mkl/BUILD:203:22: Illegal ambiguous match on configurable attribute "deps" in //tensorflow/core/kernels/mkl:mkl_qmatmul_op:
//tensorflow/tsl:linux_x86_64_with_onednn_v3
//tensorflow/tsl:linux_x86_64
Multiple matches are not allowed unless one is unambiguously more specialized.
```
So it seems like we do need the `build_with_onednn_v2` config.